tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -41,6 +41,14 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_read_test_pem(S2N_DEFAULT_TEST_PRIVATE_KEY, private_key, S2N_MAX_TEST_PEM_SIZE));
         EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key(conn->config, cert_chain, private_key));
 
+        uint16_t expected_count = S2N_CIPHER_SUITE_COUNT;
+        if (s2n_is_in_fips_mode()) {
+            EXPECT_SUCCESS(s2n_connection_set_cipher_preferences(conn, "test_all_fips"));
+            expected_count = S2N_FIPS_CIPHER_SUITE_COUNT;
+        } else {
+            EXPECT_SUCCESS(s2n_connection_set_cipher_preferences(conn, "test_all"));
```

Comment:
Do we want to use `test_all_tls13` instead as we are testing tls1.3 path?

Reply:
This isn't TLS13 specific, just not-fips. If I use test_all_tls13, a cipher is chosen which isn't in the default pref list.